### ISSUE TITLE
On Hold - Format numbers with selected language

### ISSUE THREAD

**Issue by KNXBroker** [JonahKr/power-distribution-card#39]
Hello,

since HA Version 0.118 the format of numbers is according to the selected language, pull request [#7516](https://github.com/home-assistant/frontend/pull/7516) and [#7763](https://github.com/home-assistant/frontend/pull/7763)

It would be great if the numbers of this card would be formated the same way. Best regards

![Unbenannt](https://user-images.githubusercontent.com/25910703/102246004-89d53a00-3efe-11eb-8b78-64fa514044cf.png)

**Comment by JonahKr** [comment]
Seems like it wouldn't be that difficult to implement. I need to check however if the endpoint for this is exposed. I will research a bit and will update on this topic.

**Comment by JonahKr** [comment]
The related Hass file: https://github.com/home-assistant/frontend/blob/dev/src/common/string/format_number.ts

**Comment by JonahKr** [comment]
I added it in https://github.com/JonahKr/power-distribution-card/commit/3fa17a7a42d0e53583742c19757c14a0dea12c7d and it will be included in v2.0

**Comment by KNXBroker** [comment]
Hi @JonahKr , your reactivity is amazing!! Hopefully it was not too fast this time... 😃:

I just saw that formating the numbers will be an LOVELACE option in the very near future:  https://github.com/home-assistant/frontend/pull/7925
Could you please check if you could support this as well? I think a lot of user had not been happy with the new formatting.

Best regards

**Comment by JonahKr** [comment]
Good evening @KNXBroker , oder auch einen wundnerschönen guten Abend 😉 
 I just read through the Issue wich is mentioned in the PR (https://github.com/home-assistant/frontend/issues/7745) and the PR itsself.
 So looking at the later, i don't think the PR is ready for another month and considering the size aswell as the discussion about this, will take at least 2 additional months to get through review. How ever i understand that the formatting might irritate people.
I have 2 Options now: Leave it implemented or remove it again and just wait on how it develops on the Core/Frontend side.
For now i will just let you decide if you want because neither is a lot of work. 

Anyways. Thanks for the info and i will recheck this in a month or so.

**Comment by KNXBroker** [comment]
Guten Morgen @JonahKr ,

I saw this while I wanted to adapt another custom card with this functionality. I realized as well, that the formutNumber() function cannot be imported and has to be copied. Don't know if this make sense. 

Maybe you should remove the PR and wait how things will develop? I'm no specialist, but the formatNumber() should be made availabe by LOVELACE natively? What do you thing, is this actually possible? I would write a LOVELACE issue with your feedback.

Viele Grüße, KNXBroker

**Comment by JonahKr** [comment]
Ok so i removed this for now and will take a look at it in a month again or smth.
I will just keep this open so i don't forget it.

**Comment by ferdinandovivacqua** [comment]
Hi, what is the status of this issue? can we come back to the previous (<0.118 release)? Now we have number above 1000 with thousand without any possibility to change this format.

**Comment by JonahKr** [comment]
Atm I don't have formatting implemented. I wanted to wait on what the homeassistant devs decide on. Haven't revisited it though. Do you have any new infos @ferdinandovivacqua ?

**Comment by ferdinandovivacqua** [comment]
> Atm I don't have formatting implemented. I wanted to wait on what the homeassistant devs decide on. Haven't revisited it though. Do you have any new infos @ferdinandovivacqua ?

No. Hoping this annoying regression - I'd say - will be fixed soon

**Comment by JonahKr** [comment]
**As a update:**
I added the formatNumber function to CustomCard helpers:
https://github.com/custom-cards/custom-card-helpers/releases/tag/1.6.7
I firmly believe that this should be standardized amongst most custom-cards possible and has to be decided/changeable from homeassistant frontend. I will try to update the function there with the development made in the frontend repo. (with delay)

Still haven't implemented it into the Card yet though.

**Comment by JonahKr** [comment]
*Another update:*
the PR https://github.com/home-assistant/frontend/pull/7925 adds this as a new setting much like the language one to the frontend. Therefore i will, once the cch lib is updated, integrate this feature.

**Comment by JonahKr** [comment]
Since most participants in this thread are kind off invested in this topic, feel free to participate in the PR adding the new Changes to CCH. https://github.com/custom-cards/custom-card-helpers/pull/30

**Comment by JonahKr** [comment]
Long time no see 👋 
My PR in CCH got merged and thus i added it in the latest commit: https://github.com/JonahKr/power-distribution-card/commit/ea673032f45e2f779913368fe382f1c9aff129d2
I will include it in the next release in the next few weeks.

Take Care 👍
